### ISSUE TITLE
questions about the implementation result

### ISSUE THREAD

**Issue by zwei-lin** [feihuzhang/SeparableFlow#6]
Thanks for the great code！
I have tried to reproduce the validation result on Sintel after the training on C+T by running run.sh. However, I got 1.6 on Sintel clean and 2.8 on Sintel final, which are worse than those reported in the paper.  Besides, I download the pretrained model of things provided by you, and the validation result is 1.3 on Sintel clean. 
I am confused now. Is the checkpoint of the pretrained model on C+T obtained by run.sh? or maybe there is something wrong with run.sh? I just follow the README.md to run the code, but don't get good results.
Looking forward to your reply！

**Comment by feihuzhang** [comment]
That might be caused by insufficient training. The results on C+T should be close to 1.24-1.35 on Sintel clean and 2.55-2.7 on Sintel final. I have updated the train.py and resolve the bug with the balance weights of the loss function.

**Comment by VDIGPKU** [comment]
Hi feihu,
Thanks for your reply!
I train the model on C+T training set with the new code. However, I got 1.68 on Sintel clean and 3.15 on Sintel final, which is worse than before. I don't know why it happens. Do you get a better result with the new code?

**Comment by zwei-lin** [comment]
Hi feihu,
Thanks for your reply!
I train the model on C+T training set with the new code. However, I got 1.68 on Sintel clean and 3.15 on Sintel final, which is worse than before. I don't know why it happens. Do you get a better result with the new code?
